app/Models/Beatmapset.php

```diff
@@ -1319,4 +1326,21 @@ public static function removeMetadataText($text)
 
         return preg_replace($pattern, '', $text);
     }
+
+    private static function searchCacheKey(array $params)
+    {
+        ksort($params);
+        unset($params['user']);
+
+        return 'beatmapset-search:'.serialize($params);
+    }
+
+    private static function shouldCacheSearch(array $params)
+    {
+        return !(
+            present($params['query'])
+            || in_array($params['status'], [2, 6], true) // favourites, ranked.
```

Comment:
More like 6 is own maps. Ranked is another parameter :dancer: 

Accidentally commented in somewhere useless :ok_hand: